tests/slsa_analyzer/checks/test_registry.py

```diff
@@ -213,7 +193,6 @@ def test_prepare_successfully(self) -> None:
         # at least one runner initialized and no check circular
```

Comment:
I think we can remove `at least one runner initialized` here.

Reply:
Fixed in 477c99b8929876b4ba6d5766619d266c78d095fb.